source/stylesheets/modules/_nav.scss

```diff
@@ -0,0 +1,10 @@
+nav {
+  ul {
+    li {
+      display: inline;
+      float: right;
```

Comment:
How about:

```css
nav {
  display: flex;
  flex-direction: column;
}

ul {
  align-self: flex-end;
   display: flex;
}
```

Then you can put the `li` tags in the regular order in the source too.

Reply:
I have nested the nav with the ul and li element, like that 0ad6d4c - without the scope to the `nav` all `ul` items would receive the suggestion.